readthedocs/builds/tasks.py

```diff
@@ -453,3 +457,150 @@ def send_build_status(build_pk, commit, status, link_to_build=False):
             build.project.slug
         )
         return False
+
+
+@app.task(queue='web')
+def send_build_notifications(version_pk, build_pk, event):
+    version = Version.objects.get_object_or_log(pk=version_pk)
+
+    if not version or version.type == EXTERNAL:
+        return
```

Comment:
I think we probably want to send webhook notifications for external versions.

Reply:
That could be a future option, I think for now that would be too spammy.